docs/layouts/shortcodes/endpoint-description.md

```diff
@@ -35,9 +35,13 @@ Or the `redis-persistent` type for persistent Redis:
 
 Persistent Redis requires a disk to store data.
 {{ else if eq $type "network-storage" }}
-You can define `<SERVICE_NAME>` as you like, but it should not include underscores (`_`).
+You can define `<SERVICE_NAME>` as you like, but it shouldn't include underscores (`_`).
 {{ end }}
 
```

Comment:
:hammer_and_wrench:

Although we do not want to over-use notes, I think there might be a good use case for that, given the potential for loss of data.
```suggestion
{{< note theme="warning">}} ... {{< /note >}}
```
It's probably being overly cautious since customers won't probably be re-checking the documentation before renaming a service, but if they remember there were some warnings on the first setup, it increases the likeliness that they'll re-open the page.

Reply:
I definitely considered a note, but decided against it. Partly because, as I mentioned, I think it's disruptive of the flow at a time when people are getting started with the service. I think it could create a worse experience -- more fears, less getting things done -- at a time when they are uncertain. I think a small note without too much attention is the right fit during that part of the task.

On the other hand, I've seen some evidence that years of ads being separate from text on websites have trained people to pay *less* attention to text in boxes like that. :thinking: